packages/matter-node.js/src/storage/StorageBackendDisk.ts

```diff
@@ -27,19 +27,21 @@ export class StorageBackendDisk implements Storage {
         // nothing to do
     }
 
-    buildStorageKey(context: string, key: string): string {
-        return context + "." + key;
+    buildStorageKey(contexts: string[], key: string): string {
+        const contextKey = contexts.join('.');
+        if (!key.length || !contextKey.length || contextKey.includes("..") || contextKey.startsWith(".") || contextKey.endsWith(".")) throw new Error("Context must not be an empty string!");
```

Comment:
Would be good if error message matched test.  Bonus if it calls out the specific element that fails.

Also not sure about having "!" at the end of message unless that's a formal convention?

Reply:
In fact it ends  up in "one of the context keys was an empty string" when that matches